.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "next/core-web-vitals"
+  }
```

Comment:
can you use biome here

Reply:
you can keep this file if you want, but we should use biome so that all tscircuit projects have somewhat consistent conventions